sample.env

```diff
@@ -4,8 +4,8 @@
 # There are a number of profiles available to start up groups of services. 
 # Additionally, each individual service in this project can be started by specifying its service name as a profile.
 # The currently available profile groups are listed below. 
-# basic, webapp, intersection, intersection_no_api, conflictmonitor, addons, obu_ota
-COMPOSE_PROFILES=basic,webapp,intersection
+# basic, webapp, kafka_full, intersection, intersection_no_api, conflictmonitor, addons, obu_ota
+COMPOSE_PROFILES=basic,webapp,kafka_full,intersection
```

Comment:
Note: Rather than assigning the intersection profile to the jpo-utils Kafka service, I've included kafka_full in the default Compose profiles by default.